--- popup.c.str	2009-04-13 08:44:56.000000000 +0200
+++ popup.c	2009-04-13 08:45:07.000000000 +0200
@@ -508,6 +508,6 @@
 	PopupError((Widget)NULL, ptr); 
 	allowPopup = True;
     } else {
-	fprintf(stderr, ptr);
+	fprintf(stderr, "%s", ptr);
     }
 }
--- xmh.h.str	2009-04-13 08:46:06.000000000 +0200
+++ xmh.h	2009-04-13 08:46:22.000000000 +0200
@@ -66,7 +66,7 @@
 
 #define DEBUG(msg) \
 	if (app_resources.debug) \
-	    {(void)fprintf(stderr, msg); (void)fflush(stderr);}
+	    {(void)fprintf(stderr, "%s", msg); (void)fflush(stderr);}
 
 #define DEBUG1(msg, arg) \
 	if (app_resources.debug) \
